lib/manager/components/FeedSourceTableRow.js

```diff
@@ -185,11 +185,12 @@ export default class FeedSourceTableRow extends PureComponent<Props> {
 
     // data for latest validation columns
     const currentVersionData = this.getVersionDisplayData(feedSource.latestValidation)
+    const latestVersionId = feedSource.latestValidation && feedSource.latestValidation.feedVersionId
```

Comment:
Can we do the same for `comparisonValidationSummary && comparisonValidationSummary.feedVersionId`?